src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1

```diff
@@ -295,6 +295,62 @@ Test-CreateAndGetSoftwareUpdateConfigurationWithAzureDynamicGroupsOnlyWithOutTag
                                        -StartTime $startTime `
                                        -ForUpdate
 
+$query1Scope = @(
+        "/subscriptions/cd45f23b-b832-4fa4-a434-1bf7e6f14a5a/resourceGroups/mms-wcus"
+    )
+
+    $query1Location =@("Japan East", "UK South")
```

Comment:
Shoudl not hard code locations , but use resource manager APIs to find available locations, and select from that list (you cna prefer a location, if it exists)

Reply:
@markcowl , Thank you for the feedback. Can you give me any doc location for the API usage that help to find the available locations? I couldn't easily locate the API. any hint will be appreciated